docs/install/kubernetes.md

```diff
@@ -0,0 +1,152 @@
+## Requirements
+
+Before proceeding, please ensure that you have both Helm 3.5+ and the
+[latest version of Coder](https://github.com/coder/coder/releases) installed.
+You will also need to have a Kubernetes cluster running K8s 1.19+.
+
+## Install Coder with Helm
+
+> **Warning**: Helm support is new and not yet complete. There may be changes
+> to the Helm chart between releases which require manual values updates. Please
+> file an issue if you run into any issues.
+
+1. Create a namespace for Coder, such as `coder`:
+
+   ```console
+   $ kubectl create namespace coder
+   ```
+
+1. Create a PostgreSQL deployment. Coder does not manage a database server for
+   you.
+
+   - If you're in a public cloud such as
+     [Google Cloud](https://cloud.google.com/sql/docs/postgres/),
+     [AWS](https://aws.amazon.com/rds/postgresql/),
+     [Azure](https://docs.microsoft.com/en-us/azure/postgresql/), or
+     [DigitalOcean](https://www.digitalocean.com/products/managed-databases-postgresql),
+     you can use the managed PostgreSQL offerings they provide. Make sure that
+     the PostgreSQL service is running and accessible from your cluster. It
+     should be in the same network, same project, etc.
+
+   - You can install Postgres manually on your cluster using the
+     [Bitnami PostgreSQL Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/postgresql#readme). There are some
+     [helpful guides](https://phoenixnap.com/kb/postgresql-kubernetes) on the
+     internet that explain sensible configurations for this chart. Example:
+
+     ```sh
+     # Install PostgreSQL
+     helm repo add bitnami https://charts.bitnami.com/bitnami
+     helm install coder-db bitnami/postgresql \
+         --namespace coder \
+         --set auth.username=coder \
+         --set auth.password=coder \
+         --set auth.database=coder \
+         --set persistence.size=10Gi
+     ```
+
+     The cluster-internal DB URL for the above database is:
+
+     ```
+     postgres://coder:coder@postgres-postgresql.coder.svc.cluster.local:5432/coder?sslmode=disable
+     ```
+
+     > Ensure you set up periodic backups so you don't lose data.
+
+   - You can use
+     [Postgres operator](https://github.com/zalando/postgres-operator) to
+     manage PostgreSQL deployments on your Kubernetes cluster.
+
+1. Download the latest `coder_helm` package from
+   [GitHub releases](https://github.com/coder/coder/releases).
+
+1. Create a secret with the database URL:
+
+   ```sh
+   # Uses Bitnami PostgreSQL example. If you have another database,
+   # change to the proper URL.
+   kubectl create secret generic coder-db-url -n coder \
+      --from-literal=url=postgres://coder:coder@postgres-postgresql.coder.svc.cluster.local:5432/coder?sslmode=disable
+   ```
```

Comment:
@deansheather I added some docs on creating the secret for easy reference. I did some research, but wanted to check:

are there any immediate red flags with using `--from-literal` here?

Reply:
You'll want to quote like `--from-literal url="xyz"` because the question mark will mess up the command. Other than that if it works then it works